test/integration/multi_clients_test.exs

```diff
@@ -0,0 +1,122 @@
+defmodule Integration.TokenProvider.MultiClientsTest do
```

Comment:
Should there also be a test that verifies the `:client` option actually configures the default client?

Reply:
The [original encryption test](https://github.com/primait/auth0_ex/blob/PLATFORM-1219/task/allow-multiple-tenants-to-be-configured/test/integration/encrypted_redis_token_cache_test.exs) still uses the `:client` option, but maybe it's too implicit?